src/modules/settings/components/settings/twitch/ScrollVideoSeek.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Panel from 'rsuite/Panel';
+import Toggle from 'rsuite/Toggle';
+import {registerComponent} from '../../Store.jsx';
+import {CategoryTypes, SettingIds} from '../../../../../constants.js';
+import styles from '../../../styles/header.module.css';
+import useStorageState from '../../../../../common/hooks/StorageState.jsx';
+
+function ScrollVideoSeek() {
+  const [value, setValue] = useStorageState(SettingIds.SCROLL_VIDEO_SEEK);
+
+  return (
+    <Panel header="Seek by scrolling">
+      <div className={styles.toggle}>
+        <p className={styles.description}>Enables alt+scrolling on the twitch player to seek through VODs</p>
```

Comment:
this + title copy needs better casing + formatting

Reply:
I've checked other settings and I think the casing should be correct now 👍